test/rsm/test.ts

```diff
@@ -1,44 +1,36 @@
 // See README.md for prerequisites for this to run
 
-const { Orchestrator, Config } = require('../../src')
-
-const testDna = Config.dna("test.dna.gz")
-
-const config = Config.gen({
-  tester: testDna,
-})
+import { Orchestrator, Config, InstallHapps } from "../../src"
+import { ScenarioApi } from "../../src/api"
+import path from 'path'
 
 const orchestrator = new Orchestrator()
 
-orchestrator.registerScenario('list dnas', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dnas = await antony.admin().listDnas()
-  console.log('dnas', dnas)
-
-  t.equal(dnas.length, 1)
-})
-
-orchestrator.registerScenario('call zome', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const result = await antony.call('tester', 'foo', 'foo', { anything: 'goes' })
-  console.log('result', result)
-
-  t.equal(result, 'foo')
-})
-
-orchestrator.registerScenario('state dump', async (s, t) => {
-  const { antony } = await s.players({ antony: config })
-  await antony.spawn()
-
-  const dump = await antony.stateDump('tester')
-  t.equal(dump.length, 3)
-  t.ok(typeof dump[0].element === 'object')
-  t.ok(typeof dump[1].element === 'object')
-  t.ok(typeof dump[2].element === 'object')
+const conductor1Config = Config.gen()
+const conductor2Config = Config.gen()
+
+const conductor2Apps: InstallHapps = [
+  // agent 1
+  [
+    // dna 1
+    path.join(__dirname, 'test.dna.gz')//,
+    // dna 2...
+  ],
+  // agent 2
+  [
+    // dna 1
+    path.join(__dirname, 'test.dna.gz')//,
+    // dna 2...
+  ]
+]
+
+orchestrator.registerScenario('list dnas', async (s: ScenarioApi, t) => {
+  const [[antonyConductor, [[antony1TestCell], [antony2TestCell]]]] = await s.players(
+    [[conductor2Config, conductor2Apps]]
+  )
+  const res = await antony1TestCell.call('foo', 'foo', null)
+  // console.log('result!', res)
+  t.equal(res, 'foo')
 })
```

Comment:
this is core to the changes, shows the API

note that the destructuring doesn't have to happen all at once, you can store variables at different levels, call them in different ways. 

The key is having simultaneous access to the conductor, as well as direct access to the cells. Makes using `.call` a no-brainer

Reply:
The key to this API is that the output/result of 
s.players is a mirror of the input. 

Also @zippy I made it so that you can spawn/initialize apps on a per-conductor basis, rather than all-or-nothing